### PR TITLE
Bug fix for irregularly shaped lattices

### PR DESCRIPTION
**Pull request by wbinventor**
This PR is a bug fix for issue #135. The new implementation allows for the creation of ``Lattices`` with different numbers of ``Universes`` along and x, y dimensions. In addition, the x,y width (or pitch) may now be different for the x and y domains.

### CODE REVIEW

File: src/Universe.cpp [hunk 1]
```diff
@@ -1079,11 +1079,11 @@ void Lattice::setWidth(double width_x, double width_y) {
...
- * @param num_x the number of Lattice cells along x
  * @param num_y the number of Lattice cells along y
+ * @param num_x the number of Lattice cells along x
  * @param universes the array of Universes for each Lattice cell
  */
-void Lattice::setUniverses(int num_x, int num_y, Universe** universes) {
+void Lattice::setUniverses(int num_y, int num_x, Universe** universes) {
 
   /* Clear any Universes in the Lattice (from a previous run) */
   for (int i=0; i < _num_x; i++)
```
Comment (samuelshaner) [inline src/Universe.cpp]: I think this should be `_num_y` to be consistent with lines 804-805.

File: src/Universe.cpp [hunk 2]
```diff
@@ -1104,7 +1104,7 @@ void Lattice::setUniverses(int num_x, int num_y, Universe** universes) {
 
     _universes.push_back(std::vector< std::pair<int, Universe*> >());
 
-    for (int j = 0; j< _num_x; j++){
+    for (int j = 0; j < _num_x; j++){
```
Comment (samuelshaner) [inline src/Universe.cpp]: I feel like having `i` correspond to the x index and `j` correspond to the y index would be more conventional.

File: src/Universe.cpp (hunk 1 above)
Comment (wbinventor) [inline src/Universe.cpp]: Very good catch.

File: src/Universe.cpp (hunk 2 above)
Comment (wbinventor) [inline src/Universe.cpp]: I agree and pushed the revised loop counters.

**Pull request merged**